libs/libmysqlxx/cmake/find_mysqlclient.cmake

```diff
@@ -1,6 +1,6 @@
 option(ENABLE_MYSQL "Enable MySQL" ${ENABLE_LIBRARIES})
 if(ENABLE_MYSQL)
-    if(OS_LINUX)
+    if(OS_LINUX OR APPLE)
```

Comment:
But it breaks build: https://clickhouse-builds.s3.yandex.net/7621/aa09bcb3bcb1ce190b4723aedd896cebbf632b05/build_log_541663648_1572896163.txt